contracts/Strategy.sol

```diff
@@ -235,7 +237,6 @@ contract Strategy is AccessControl, ERC20Rewards, StrategyMigrator { // TODO: I'
             }
 
         } catch {
-            delete cached;
```

Comment:
is this correct that we don't delete `cached` anymore? @alcueca

Reply:
`cached` doesn't exist, now we have `baseCached` and `poolCached`. We delete `poolCached` in L227, and `baseCached` starts this function call at zero because we are `INVESTED`. After this call, `poolCached` will be zero, and `baseCached` and `fyTokenCached` might be above zero depending on the result of the call (did we eject? did the pool have fyToken? did we drain?)